src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -225,6 +225,8 @@ static UnitAttachment get(final UnitType type, final String nameOfAttachment) {
   private ArrayList<String> m_receivesAbilityWhenWith = new ArrayList<>();
   // currently used for: placement in original territories only
   private HashSet<String> m_special = new HashSet<>();
+  // Manually set TUV
+  private int m_tuv = -1;
```

Comment:
I'm not 100% sure about this, but when loading an old save game, I think this will get set to the default value of `0` since `m_tuv` doesn't exist in the serialization stream.  If that's true, is it going to cause a problem?

**EDIT:** After reviewing more code, I may have answered my own question.  It looks like in `TuvUtils` that you only use this value if it's greater than zero, so that both `-1` and `0` are handled identically.  True?

Also, the `m_` prefix is a Checkstyle violation.  Not sure if you just want to keep the field names in this class consistent.  If so, you'll need to bump the warning count in `gradle.properties`.  (I believe I modified `ValidateAttachmentsTest` correctly so it doesn't fail if you do not use the `m_` prefix; please let me know if that's not the case.)

Reply:
Exactly, I realized the same thing on the default value. It would be better to eventually change it to be greater than -1 in case there are any cases map makers want tuv=0. I'd prefer to keep it consistent for now and change them all at once.